self-tests-staging.yml

```diff
@@ -0,0 +1,37 @@
+include:
+  - filename: self-tests.yml
+
+containers:
+  - name: evg-container
+    working_dir: /
+    image: "hadjri/evg-e2e-test-ubuntu"
+    resources:
+      cpu: 4096
+      memory_mb: 8192
+    system:
+      cpu_architecture: x86_64
+      operating_system: linux
+
+buildvariants:
+  - name: ubuntu1604-container
+    display_name: Ubuntu 16.04 (Container)
+    run_on:
+      - evg-container
+    expansions:
+      goos: linux
+      goarch: amd64
+      IS_DOCKER: true
+      GOROOT: /usr/local/go
+      mongodb_url: https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1604-4.2.18.tgz
+      decompress: tar zxvf
+      notary_client_url: https://macos-notary-1628249594.s3.amazonaws.com/releases/client/v3.1.2/linux_amd64.zip
```

Comment:
I think we can remove `notary_client_url` and `sign_macos` because the container doesn't have the MacOS signing service installed on it.